DualView/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <application>
         <activity
+            android:hardwareAccelerated="false"
```

Comment:
Why do you need this?

Reply:
Because the sample works with a large bitmap and needs this to avoid the bellow crash:

    java.lang.RuntimeException: Canvas: trying to draw too large(228352000bytes) bitmap.